src/components/DBEntryManagerPanel/AddOrEditDBEntry/index.tsx

```diff
@@ -0,0 +1,614 @@
+            for (let j = 0; j < value.length; j += 1) {
+              if (field.minNumber && value[j] < field.minNumber) {
+                validationError = `${field.label} values must be at least ${field.minNumber}`;
+                break;
+              } else if (field.maxNumber && value[j] > field.maxNumber) {
+                validationError = `${field.label} values must be at most ${field.maxNumber}`;
+                break;
+              }
+            }
+          }
+        }
+      }
+    }
+
+    const validationFailed = !!validationError;
+
+    const renderEntryField = (field: DBEntryField, disabled: boolean) => {
+      if (field.type === DBEntryFieldType.String) {
+        if (field.choices) {
+          return (
+            <div
+              key={field.objectKey}
+              className="mb-2"
+            >
+              <div className="input-group"
+                style={{
+                  pointerEvents:
+                    (disabled) ? 'none' : 'auto',
+                }}
+              >
+                <ButtonInputGroup
+                  label={field.label}
+                >
+                  {
+                    field.choices.map((choice) => {
+                      return (
+                        <RadioButton
+                          key={choice.value}
+                          text={choice.title}
+                          selected={entry[field.objectKey] === choice.value}
+                          onSelected={() => {
+                            entry[field.objectKey] = choice.value;
+                            dispatch({
+                              type: ActionType.UpdateDBEntry,
+                              DBEntry: entry,
+                            });
+                          }}
+                          ariaLabel={choice.title}
+                        />
+                      );
+                    })
+                  }
+                </ButtonInputGroup>
+
+              </div>
+            </div>
+          );
+        }
+
+        return (
+          <div
+            className="mb-2"
+            key={field.objectKey}
+          >
+            <div className="input-group">
+              <span
+                className="AddDBEntry-input-label input-group-text"
+                id="AddDBEntry-form-name-label"
+              >
+                {field.label}
+              </span>
+              <input
+                id="AddDBEntry-form-name-input"
+                disabled={disabled}
+                type="text"
+                className="form-control"
+                placeholder={field.placeholder}
+                aria-describedby="AddDBEntry-form-name-label"
+                value={field.label === INPUT_PLACEHOLDER ? '' : entry[field.objectKey] || ''}
+                onChange={(e) => {
+                  entry[field.objectKey] = (
+                    e.target.value
+                      .replace(/[^a-zA-Z0-9\s(),-]/g, '')
```

Comment:
Is this always the case that we want to apply those filters? Probably someone will want to add a string that contains special chars